jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/test/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListPresenterTest.java

```diff
@@ -341,6 +343,48 @@ public void testViewProcessInstanceActionCondition() {
     }
 
     @Test
+    public void testGoTaskErrorSummary() {
+        doAnswer(new PerspectiveAnswer(TASKS_ADMIN)).when(authorizationManager).authorize(any(ResourceRef.class),
+                                                                                          eq(identity));
+
+        ExecutionErrorSummary errorSummary =
+                ExecutionErrorSummary.builder()
+                        .activityId(Long.valueOf("1"))
+                        .build();
+
+        presenter.goToTask(errorSummary);
+
+        final ArgumentCaptor<PlaceRequest> captor = ArgumentCaptor.forClass(PlaceRequest.class);
+        verify(placeManager).goTo(captor.capture());
+        final PlaceRequest request = captor.getValue();
+        assertEquals(TASKS_ADMIN,
+                     request.getIdentifier());
+        assertEquals(errorSummary.getActivityId().toString(),
+                     request.getParameter(PerspectiveIds.SEARCH_PARAMETER_TASK_ID,
+                                          null));
+    }
+
+    @Test
+    public void testViewTaskActionCondition() {
+        doAnswer(new PerspectiveAnswer(TASKS_ADMIN)).when(authorizationManager).authorize(any(ResourceRef.class),
+                                                                                                eq(identity));
+
+        assertTrue(presenter.getViewTaskActionCondition().test(ExecutionErrorSummary.builder()
+                                                                       .type(ExecutionErrorType.TASK)
+                                                                       .activityId(Long.valueOf(1)).build()));
+
+        when(authorizationManager.authorize(any(ResourceRef.class),
```

Comment:
@nmirasch because the authorization is denied from this point on, it invalidates the other tests you have like a different error type or error without `activityId`. You could either move those two tests before this point or split that in another test method.